subsys/bluetooth/host/conn.c

```diff
@@ -1096,7 +1097,10 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 	switch (conn->state) {
 	case BT_CONN_CONNECTED:
 		if (conn->type == BT_CONN_TYPE_SCO) {
-			/* TODO: Notify sco connected */
+			/* Notify sco connected */
```

Comment:
nit: You probably don't need the comment anymore.

Reply:
Removed